modules/_labs/combobox/react/lib/Combobox.tsx

```diff
@@ -33,7 +33,7 @@ export interface ComboboxProps extends GrowthBehavior, React.HTMLAttributes<HTML
    * The `aria-label` for the Combobox clear button.
    * @default Reset Search Input
    */
-  clearButtonLabel: string;
+  clearButtonLabelAriaLabel?: string;
```

Comment:
🔴  Note: We can do this since it's a labs component, but we need to message that this is a breaking change when we release.

Reply:
If we want to make this change, I'm not sure it should happen in this PR anyway